pgjdbc/src/main/java/org/postgresql/replication/PGReplicationStream.java

```diff
@@ -21,8 +21,7 @@
  * disconnect by timeout from server.
  */
 public interface PGReplicationStream
-    extends AutoCloseable
-    /* hi, checkstyle */ {
+    extends AutoCloseable { 
```

Comment:
Would you please try removing whitespace after the opnening {  ?

Reply:
That's what I thought it was as well, seems rather picky ?

Reply:
ah, got it

Reply:
this has been done